src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs

```diff
@@ -193,13 +190,15 @@
                          "methodName" (get-in props [:method "name"])
                          "methodVersion" (get-in props [:method "snapshotId"]))
               :headers utils/content-type=json
-              :on-done (fn [{:keys [success? get-parsed-response status-text]}]
-                         (if-not success?
-                           (swap! state assoc :error status-text :blocking-text nil)
-                           (let [template (get-parsed-response false)]
+              :on-done (fn [{:keys [success? get-parsed-response]}]
+                         (let [response (get-parsed-response false)]
+                           (if-not success?
+                             (do
+                               (modal/pop-modal)
```

Comment:
There is no useful action to be done at the import screen after this error, so remove it